crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -97,7 +97,7 @@ def finished?
 
     def save
       progress_file_path.open("w") do |f|
-        f.write(JSON.pretty_generate(progress))
+        f.write(YAML.dump(progress))
```

Comment:
@MaximilianMeister Yes, sorry for the bad commit message - I was very tired when working on this and forgot to put justification. Will fix that!

@jsuchome No I don't think we should, otherwise I wouldn't have proposed this change to YAML.